pontoon/base/tests/__init__.py

```diff
@@ -100,11 +104,16 @@ class TranslationFactory(DjangoModelFactory):
     locale = SubFactory(LocaleFactory)
     string = Sequence(lambda n: 'translation {0}'.format(n))
     user = SubFactory(UserFactory)
-
     class Meta:
         model = Translation
 
 
+class IdenticalTranslationFactory(TranslationFactory):
+    @factory.sequence
+    def entity(n):
```

Comment:
Does this rely on the sequence counter being on the same number as the translation string sequence counter? What happens if they get out of sync?

Also, does this work when I want to define properties about the entity? Like:

```python
IdenticalTranslationFactory.create(
     entity__resource__project__slug='foobar'
)
```